src/atopile/front_end.py

```diff
@@ -78,7 +78,10 @@ def __repr__(self) -> str:
 @define
 class Physical(Base):
     """Let's get physical!"""
-    unit: pint.Unit
+
+    unit: pint.Unit = field(
```

Comment:
re-adding this comment; I don't think we should be casting this to a string, or, if there's a reason we should be doing so, we need to also change the type-hint.